### DISCUSSION TITLE
0x06c-Reverse-Engineering-and-Tampering - Bad link

### DISCUSSION THREAD

**Discussion by rsenet** [OWASP/mastg#2354] [Ideas]
The link in https://mas.owasp.org/MASTG/iOS/0x06c-Reverse-Engineering-and-Tampering/#corellium to Corellium article is currently wrong and should be replace by https://support.corellium.com/features/apps/testing-ios-apps

**Comment by cpholguera** [comment]
Thanks for reporting this @rsenet, I've created an issue for this

https://github.com/OWASP/owasp-mastg/issues/2357

Would you like to open a pull request to fix it? If you'd like to start contributing regularly I can invite you to the project on GitHub so we can assign you issues you'd like to work on. What do you think?

**Comment by cpholguera** [comment]
Fixed
